pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/kube_ingress_reconciler.go

```diff
@@ -146,7 +147,7 @@ func generateMetadata(isvc *v1beta1.InferenceService,
 	componentType constants.InferenceServiceComponent, name string) metav1.ObjectMeta {
 	// get annotations from isvc
 	annotations := utils.Filter(isvc.Annotations, func(key string) bool {
-		return !utils.Includes(constants.ServiceAnnotationDisallowedList, key)
+		return !utils.Includes(constants.RawServiceAnnotationDisallowedList, key)
```

Comment:
same here.
instead, we need to make the feature work with raw as well, and propose the needed changes on upstream.

Reply:
fixed